i18n/locale-et.po

```diff
@@ -3410,24 +3410,24 @@ msgstr "Haldus"
 #: src/marketplace/details/utils.ts src/marketplace/routes.js
 #: src/marketplace/sidebar.js
 msgid "Marketplace"
-msgstr "Turuplats"
+msgstr "Teenusekataloog"
```

Comment:
Changing this would need approval from RP project team as this is sitting in some documents.